src/lib/ndpi_main.c

```diff
@@ -9383,6 +9407,25 @@ const char *ndpi_confidence_get_name(ndpi_confidence_t confidence)
 
 /* ****************************************************** */
 
+const char *ndpi_fpc_confidence_get_name(ndpi_fpc_confidence_t fpc_confidence)
+{
+  switch(fpc_confidence) {
+  case NDPI_FPC_CONFIDENCE_UNKNOWN:
+    return "Unknown";
+
+  case NDPI_FPC_CONFIDENCE_IP:
+    return "IP address";
+
+  case NDPI_FPC_CONFIDENCE_DNS:
+    return "DNS";
+
+  default:
+    return NULL;
```

Comment:
Better to return something like `"n/a"` instead of a `NULL` or at least document that it may return `NULL` if not in sync.